src/style.css

```
@@ -1 +1,95 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 600;
+}
+
+:root {
+  --placeholder-color: #3D4E61;
+}
+
+* {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  padding: 0;
+  box-sizing: border-box;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.form {
+  width: 100%;
+  margin-top: 20px;
+  display: flex;
+}
+
+.form__searchbar {
+  position: relative;
+  padding: 0 62px;
+  width: 100%;
+  height: 70px;
+  border: 1px solid #E1E7ED;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+  transition: .3s;
+}
+
+.form__searchbar--small {
+  padding: 0 33px;
+  height: 42px;
+}
+
+.form__searchbar::after{
+  content: '';
+  display: block;
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
```

can be a bit simplify
```suggestion
  background: url(./images/Search.svg) no-repeat contain;
```